From f6c64f3dcb6b8496daa6b23207195a90dfe309c1 Mon Sep 17 00:00:00 2001
From: Dragos Bogdan <dragos.bogdan@analog.com>
Date: Tue, 31 Mar 2015 11:18:26 +0300
Subject: [PATCH 12/16] drivers/media/i2c/adv7604: Enable Video Adjustment
 controls.

The bit selects whether or not the color controls (contrast, saturation, brightness and hue) feature is enabled.
---
 drivers/media/i2c/adv7604.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/i2c/adv7604.c b/drivers/media/i2c/adv7604.c
index b05fa8c19544..b271a243a2fa 100644
--- a/drivers/media/i2c/adv7604.c
+++ b/drivers/media/i2c/adv7604.c
@@ -1820,7 +1820,7 @@ static void select_input(struct v4l2_subdev *sd)
 			afe_write(sd, 0xc8, 0x40); /* phase control */
 		}
 
-		cp_write(sd, 0x3e, 0x00); /* CP core pre-gain control */
+		cp_write(sd, 0x3e, 0x80); /* CP core pre-gain control */
 		cp_write(sd, 0xc3, 0x39); /* CP coast control. Graphics mode */
 		cp_write(sd, 0x40, 0x80); /* CP core pre-gain control. Graphics mode */
 	} else {
-- 
2.17.1

